### DISCUSSION TITLE
Uninstall Timelord

### DISCUSSION THREAD

**Discussion by nomadengineer** [Chia-Network/chia-blockchain#6893] [Support]
How do I uninstall a timelord installation from my main node machine?

**Comment by loppefaaret** [comment]
assuming ubuntu:
Close down Chia, exit venv, removing the venv folder, and doing install script again:
```
chia stop all -d
deactivate
cd ~/chia-blockchain
rm -rf venv/
sh install.sh
```
if you need the gui as well:
```
. ./activate
sh install-gui.sh
cd chia-blockchain-gui
npm run electron &
```

**Comment by BrandtH22** [comment]
@nomadengineer , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
